src/bundle/Resources/views/themes/admin/content/widget/content_create.html.twig

```diff
@@ -23,7 +23,7 @@
             <div class="ibexa-extra-actions__section-content ibexa-extra-actions__section-content--content-type">
                 <div class="ez-instant-filter">
                     <div class="ez-instant-filter__input-wrapper">
-                        <input type="text" class="ez-instant-filter__input form-control" placeholder="{{ ('Type to refine') }}" value="" />
+                        <input type="text" class="ez-instant-filter__input ibexa-input ibexa-input--text form-control" placeholder="{{ ('Type to refine') }}" value="" />
```

Comment:
I know it's not yours, but it would be nice to add translation to this placeholder